src/acl/Checklist.h

```diff
@@ -228,7 +228,7 @@ class ACLChecklist
     bool asyncCaller_; ///< whether the caller supports async/slow ACLs
     bool occupied_; ///< whether a check (fast or non-blocking) is in progress
     bool finished_;
-    allow_t allow_;
+    Acl::Answer allow_;
```

Comment:
I wasn't sure, but maybe the variable name should also be renamed?

maybe something like this?
```
Acl::Answer answer;
```

Reply:
Yes please do.

Reply:
... but with an underscore suffix to match the existing naming scheme for private data members: `answer_`.